### DISCUSSION TITLE
There is a player not visible in the HeroesProfile match results

### DISCUSSION THREAD

**Discussion by asdf1234dvcs** [Heroes-Profile/heroesprofile#1049] [Q&A]
https://www.heroesprofile.com/Match/Single/54522672

I can't see the player information for Phoenix in that match.

I understand that in the past, map hack users were hidden on the HeroesProfile.

I wonder if they are still hiding players in the same way.

**Comment by Zemill** [comment]
Either this is a banned player (on Heroes Profile), or a player who has set their account to private.  We currently do not reference map hackers on the website as we have found the method to track it to be inconsistent
